rainier-core/src/main/scala/com/stripe/rainier/core/SBCModel.scala

```diff
@@ -0,0 +1,48 @@
+package com.stripe.rainier.core
+
+import com.stripe.rainier.compute._
+import com.stripe.rainier.sampler._
+
+trait SBCModel[L, T] {
+  implicit val rng: RNG = ScalaRNG(1528673302081L)
+  def sbc: SBC[L, T]
+  def main(args: Array[String]): Unit = sbc.animate(HMC(1), 10000, 1000)
+  def samples: List[T] = sbc.posteriorSamples(goldset.size)
+  def goldset: List[T]
+  def description: String
+}
+
+object SBCUniformNormal extends SBCModel[Continuous, Double] {
+  def sbc = SBC(Uniform(0, 1))((x: Real) => Normal(x, 1))
+  def goldset =
+    List(0.770217434378013, 1.2877991075049935, 1.0761575302777342,
+      0.06063426627950119, 2.235772959187611, 1.5916139704386643,
+      0.4680530806057483, -1.3360844052727425, 0.6558428534558145,
+      0.4233930395976211)
+  def description = "Normal(x,1) with Uniform(0,1) prior"
+}
+
+object SBCLogNormal extends SBCModel[Continuous, Double] {
+  def sbc = SBC(LogNormal(0, 1))((x: Real) => LogNormal(x, x))
+  def goldset: List[Double] =
+    List(1.9444409266417129, 4.884477223783541, 3.166384786757497,
+      1.340419862750101, 62.45333008770111, 21.260796478808622,
+      2.145416458600115, 0.5978183569453146, 2.745814563529916,
+      2.326159027201714)
+  def description = "LogNormal(x,x) with LogNormal(0,1) prior"
+}
+
+/**
+  * Note: this are made-up goldsets. SBC on these is wildly slow.
```

Comment:
Hmmm we should look into that :)

Reply:
Yep! For now I left these SBCModels in, but removed them from SBCTest so that tests pass.